swift/BUILD.bazel

```diff
@@ -90,29 +89,3 @@ alias(
     name = "create-extractor-pack",
     actual = ":swift-installer",
 )
-
-# TODO: following rules are for internal repo backward compatibility only
```

Comment:
This was required to avoid having to allow to push a `codeql` PR independently of the internal repo some time ago (i.e. to maintain compatibility with what was then the internal repo `main` branch). I just forgot to clean up after this wasn't needed any more.